### ISSUE TITLE
[DataSet] Allow test ratio to zero in WildFireSplitter

### ISSUE THREAD

**Issue by MateoLostanlen** [pyronear/pyro-vision#61]
## Motivation

At the moment the input ratios of WildFireSplitter cannot be equal to zero. I would like to add this option. Indeed we are going to do a lot of tests with different extraction methods (number of frames, spacing between frames ...). As the frame extraction is long I think we will save a lot of time if we don't have to do it every time for the test set. 

I will try to propose a PR to add this option today

**Comment by x0s** [comment]
Hi Mateo,
Can you provide a snippet showing that the "the frame extraction is long" so we can measure performance, therefore improve it ?

**Comment by MateoLostanlen** [comment]
With the following code we extract 1 frame per sequence for the test set that is 220 images, on my PC the code runs in 46s
```
from pyronear.datasets.wildfire.frame_extractor import  FrameExtractor
from pathlib import Path
import timeit
path_to_wildfire = Path(r"C:\Users\mateo\Documents\Pyronear\Wildfire")
path_to_frames = Path(r"C:\Users\mateo\Google Drive\DeepLearning\Pyronear\DB\Wildfire")
path_to_states= Path(r"C:\Users\mateo\Google Drive\PyroNear\Annotation\WildFire\statesTestSet.csv")
strategy='random'
n_frames=1
start = timeit.default_timer()
frame_extractor = FrameExtractor(path_to_videos=path_to_wildfire, path_to_states=path_to_states,
                                     strategy='random',n_frames=n_frames)
labels = (frame_extractor.run(path_to_frames=path_to_frames).get_frame_labels())

stop = timeit.default_timer()

print('Time: ', stop - start)  
```

Output :

```
Writing frame labels to C:\Users\mateo\Google Drive\DeepLearning\Pyronear\DB\Wildfire\statesTestSet.labels.csv
Extracting 1 frames per state (220 in total) to C:\Users\mateo\Google Drive\DeepLearning\Pyronear\DB\Wildfire
Time:  45.90241230000001
```

**Comment by MateoLostanlen** [comment]
As requested  : https://github.com/frgfm/PyroNear/pull/63#discussion_r423622015

There is a code to test the error :

```
wildfire = WildFireDataset(metadata=metadata,path_to_frames=path_to_frames,target_names=['fire'])
ratios = {'train': 0.8, 'val': 0.2, 'test':0}

splitter =  WildFireSplitter(ratios, transforms=tf)
splitter.fit(wildfire)
print(splitter.n_samples_)

train_loader = DataLoader(splitter.train, batch_size=64, shuffle=True)
val_loader = DataLoader(splitter.val, batch_size=64, shuffle=True)
```
